test/losses_test.py

```diff
@@ -96,3 +96,23 @@ def test_entropy_cube():
 
     cube = torch.ones((nchan, npix, npix), dtype=torch.float64)
     losses.entropy(cube, 0.01)
+
+
+def test_tsv():
+    cube = torch.rand((1, 3, 3))
```

Comment:
rather than hard coding the 3 in the array creation and the 2 in the loops, how about an npix=3 and loop over (npix - 1), making it easy to extend this to larger npix values?